src/common/components/Input/index.tsx

```diff
@@ -100,17 +122,44 @@ export interface InputProps extends React.InputHTMLAttributes<HTMLInputElement>
     onBlur?: (event: React.FormEvent<HTMLInputElement> | undefined) => void;
     type?: string;
     description?: string;
+    hide?: boolean;
```

Comment:
Ought to rename this property to make more sense for what's intended, e.g. _showPasswordToggle_, _togglePassword_, _showToggle_...